### DISCUSSION TITLE
what is a keymap to open tab has closed before? like Ctrl + shift + t on vscode.

### DISCUSSION THREAD

**Discussion by yushaku** [LazyVim/LazyVim#3805] [Q&A] [answered]
at the previous version of lazyvim.  when I close a buffer by using keymap <leader>bd then I can go back to that buffer by using <ctrl + o>.
but on the latest version, I can not do it anymore.

**Comment by xzbdmw** [comment, marked as answer]
Maybe `c-6` ?

**Reply by yushaku** [reply]
oh, thanks. it works. I did not find out a document about `<c-6>`!

**Reply by xzbdmw** [reply]
That's vim builtin keymap 😄

**Comment by folke** [comment]
I think that was due to a change in Neovim, but I've since changed an option to get back the old behavior.
Can you check this is indeed the case?
And if not, what are the steps that trigger the issue?

**Reply by yushaku** [reply]
> what are the steps that trigger the issue?

i open file1, file2, file3, file4
and then i use keymap `<leader>bd` to close file4, file3, file2.

with the previous version of neovim, I can use `<C-o>` to turn back file2. do it again, and it turns back to file3, and so on...
but in the current version, it does not work anymore
with `<c-6>` it only turns back file2, although i click `<c-6>` many time.

**Reply by dpetka2001** [reply]
Update your LazyVim to latest. This should not happen any more. I tried what you said and pressing `<C-o>` correctly opens all 3 files.

**Reply by yushaku** [reply]
I think I use the latest version of nvim. which version are you using?

```bash
NVIM v0.10.0
Build type: RelWithDebInfo
```

**Reply by dpetka2001** [reply]
I said LazyVim, not Neovim.

**Reply by yushaku** [reply]
my bad
